### PR TITLE
Add features.md for feature tracking

### LINKED ISSUE #173: Add features.md for feature tracking
In order to validate each feature against a user and track it's usefulness and maturity, we would like to have an overview of all features we have built or are planning to build

### PR DESCRIPTION
**Pull request by dmsuehir**
fixes #173

Added a `features.md` document with current features, as well as upcoming features.

## Reviewer Checklist

 - [ ] Do you understand it?
 - [ ] Is it correct?
 - [ ] Is it safe?
 - [ ] Is it legally compliant?
 - [ ] Is it robust?
 - [ ] Is it simple?
 - [ ] Is it tested?
 - [ ] Is it documented?
 - [ ] Is the style consistent?

See [the reviewer guide](docs/reviews.md) for more detail on each check.

### CODE REVIEW

File: docs/features.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+# MLT Features
```
Comment (nqn) [inline docs/features.md]: Some of the reasoning behind the features file, would be to track the maturity of each feature as well. If it just landed, it being in alpha (or if it just not hardened yet).

Comment (nqn) [reply docs/features.md]: And maybe we should add a note where we add flags, that new flags should be documented here?

File: docs/features.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+|---------------------|-------------|
+| `<name>` | Name of the configuration parameter to remove. |
+
+### mlt build
+
+```
+  mlt build [--watch]
+```
+
+This command builds a local image for the current project directory.
```
Comment (nqn) [inline docs/features.md]: Maybe add the detail about the timer which is started when a file change is detected.

Comment (dmsuehir) [reply docs/features.md]: @nqn  I don't know which timer you mean...  I'm looking at the code and there's that sleep (https://github.com/IntelAI/mlt/blob/master/mlt/commands/build.py#L97) - is that what you're referring to?  I just tried it out and it seems like file changes are detected and the re-build starts as soon as I save the file.

**Pull request merged**